target/linux/ath79/dts/ar9342_pcs_cr5000.dts

```diff
@@ -18,7 +18,7 @@
 		led-upgrade = &status;
 	};
 
-	keys {
+	gpio-keys-polled {
```

Comment:
To clear this up (And yes, I should update ipq40xx and apm821xx too).
The older ePAPR and the current [DT Spec v0.2](https://github.com/devicetree-org/devicetree-specification/releases/download/v0.2/devicetree-specification-v0.2.pdf) devoted section "2.2.1 Node Names" and "2.2.2 Generic Names Recommendation" to this topic. From what I can tell, the general gist is that:
"The name of a node should be somewhat generic, reflecting the function of the device and not its precise programming model.". In 2.2.2 the spec also lists "key", "keys", "led" and "leds" as appropriate names. I would say, leave the keys and leds as they are.